src/App.css

```diff
@@ -1,38 +1,106 @@
 .App {
-  text-align: center;
+  margin-bottom: 50px auto;
+  margin-top: 0px auto;
+  background-color: #282c34;
+  min-height: 100vh;
+  clear: both;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.Ipad {
```

Comment:
I believe css class names are supposed to be `hyphen-case`

Reply:
Do you want me to change all the css class names?

Reply:
yes